packages/webpack/src/ember-webpack.ts

```diff
@@ -19,10 +19,21 @@ import {
   getAppMeta,
   getPackagerCacheDir,
   getOrCreate,
+  resolverPlugin,
+  ResolverPluginOptions,
 } from '@embroider/core';
 import { tmpdir } from '@embroider/shared-internals';
 import webpack, { Configuration, RuleSetUseItem, WebpackPluginInstance } from 'webpack';
-import { readFileSync, outputFileSync, copySync, realpathSync, Stats, statSync, readJsonSync } from 'fs-extra';
+import {
+  readFileSync,
+  outputFileSync,
+  copySync,
+  realpathSync,
+  Stats,
+  statSync,
+  readJsonSync,
+  readJSONSync,
```

Comment:
turns out `fs-extra` actually exports both of these as aliases of each other, but it's confusing to see them both imported

Reply:
Oh yeah, this is just autocomplete being silly.